portfolio/src/main/webapp/nature-walks.html

```diff
@@ -21,10 +21,10 @@ <h1>Rachel's Portfolio</h1>
         <li><a href="index.html">Home</a></li>
         <li><a href="gallery.html">Gallery</a></li>
         <li><a href="comments.html">Comments</a></li>
-        <li><a class="active" href="map.html">Map</a></li>
+        <li><a class="active" href="nature-walks.html">Canberra Nature Walks</a></li>
```

Comment:
Rather than having to maintain the list of links on every page, consider using a sidebar.html which gets included into all of them to centralise it.

Reply:
Ah ok, I was starting to wonder if there was a way to do that